pkg/analysis/passes/sourcecode/sourcecode_test.go

```diff
@@ -80,137 +80,3 @@ func TestSourceCodeNotProvidedReportAll(t *testing.T) {
 	require.Equal(t, "Source code not provided or the provided URL  does not point to a valid source code repository", interceptor.Diagnostics[0].Title)
```

Comment:
minor item, there is an extra space here between URL and "does"

Reply:
because this is a test this empty space is supposed to be the "URL". but in this test scenario there's no url because we injected the content directly.